doc/Doxyfile.in

```diff
@@ -125,7 +125,7 @@ EXAMPLE_PATTERNS       =
 EXAMPLE_RECURSIVE      = NO
 IMAGE_PATH             =
 INPUT_FILTER           =
-FILTER_PATTERNS        =
+FILTER_PATTERNS        = *.py="@FreeOrion_SOURCE_DIR@/doc/tag_filter.py"
```

Comment:
Don't expect that shebang will work because this isn't true on Windows and we can't blindly assume python2 is installed in `/usr/bin`.  I suggest you use the same pattern as used in [CMakeLists.txt:104-114](https://github.com/freeorion/freeorion/blob/f28d793be812f8a5da57bdc4db2bde3abb147f6e/CMakeLists.txt#L104-L114).

We already require python 2 in the root `CMakeLists.txt` file, so something like:

```
FILTER_PATTERNS        = *.py="@PYTHON_EXECUTABLE@ @FreeOrion_SOURCE_DIR@/doc/tag_filter.py"
```

should do the job.